pc.c

```diff
@@ -58,8 +58,8 @@ sqrt_mod_p(
 
     int ret;
 
-    if (mp_cmp_d(&g, 0) == MP_EQ){
-        mp_set(&b, 0);
+    if (mp_cmp_d(g, 0) == MP_EQ){
```

Comment:
what is the expected argument type for first arg of mp_cmp_d and mp_set ? 
Is this line covered under test now ?

Reply:
It needs the address of the mp_int variable